ddp-workspace/projects/ddp-testboston/src/app/app.module.ts

```diff
@@ -105,16 +106,18 @@ sdkConfig.tooltipIconUrl = 'assets/images/info.png';
 sdkConfig.lookupPageUrl = AppRoutes.Prism;
 sdkConfig.compositeRequiredFieldExceptions = [QuestionType.Numeric];
 sdkConfig.scrollToErrorOffset = 130;
+sdkConfig.defaultLanguageCode = 'en';
 sdkConfig.rtlLanguages = ['ar'];
 
-const initialLanguageCode = localStorage.getItem('studyLanguage') || 'en';
-
-export function translateFactory(translate: TranslateService, injector: Injector, logger: LoggingService) {
+export function translateFactory(translate: TranslateService,
+    injector: Injector,
+    logger: LoggingService,
+    language: LanguageService): () => Promise<any> {
     return () => new Promise<any>((resolve: any) => {
         const LOG_SOURCE = 'AppModule';
         const locationInitialized = injector.get(LOCATION_INITIALIZED, Promise.resolve(null));
         locationInitialized.then(() => {
-            const locale = 'en';
+            const locale = language.getAppLanguageCode();
```

Comment:
The bug occurred due to race conditions in the language setup. We should not hardcode 'en' language.